apps/web-frontend/src/locales/de.json

```diff
@@ -1,2 +1,58 @@
 {
+  "View Settings": "Einstellungen anzeigen",
+  "Planets Tonight": "Planeten heute Abend",
+  "Data Credits": "Datenquellen",
+  "Search...": "Suchen...",
+  "Observe": "Beobachten",
+  "Constellations": "Sternbilder",
+  "Constellations Art": "Kunst-Sternbilder",
+  "Atmosphere": "Atmosphäre",
+  "Landscape": "Landschaft",
+  "Azimuthal Grid": "Azimutales Gitter",
+  "Equatorial Grid": "Äquatoriales Gitter",
+  "Deep Sky Objects": "Deep-Sky-Objekte",
+  "Night Mode": "Nachtmodus",
+  "Fullscreen": "Vollbild",
+  "Loading {0}, the online Star Map": "Lade {0}, die Online-Sternkarte",
+  "Could not show the Online Star Map": "Die Online-Sternkarte konnte nicht angezeigt werden",
+  "It seems that your browser cannot load Web Assembly!": "Es scheint, dass Ihr Browser WebAssembly nicht laden kann!",
+  "Web assembly is necessary for Stellarium Web to display the star map. Please upgrade your web browser and try again!": "WebAssembly ist notwendig, damit Stellarium Web die Sternkarte anzeigen kann. Bitte aktualisieren Sie Ihren Webbrowser und versuchen Sie es erneut!",
+  "desktop version": "Desktop-Version",
+  "In the meantime, you can try the {0}!": "In der Zwischenzeit können Sie {0} ausprobieren!",
+  "Use Autolocation": "Standort automatisch ermitteln",
+  "My Locations": "Meine Standorte",
+  "Use this location": "Diesen Standort verwenden",
+  "Unknown Address": "Unbekannte Adresse",
+  "Lat {0}° Lon {1}°": "Lat {0}° Lon {1}°",
+  "Unknown": "Unbekannt",
+  "Near {0}": "In der Nähe von {0}",
+  "Planets Visibility": "Sichtbarkeit der Planeten",
+  "Night from {0} to {1}": "Nacht von {0} bis {1}",
+  "Rise": "Aufgang",
+  "Set": "Untergang",
+  "Magnitude": "Helligkeit",
+  "Distance": "Entfernung",
+  "Radius": "Radius",
+  "Spectral Type": "Spektraltyp",
+  "Size": "Größe",
+  "Ra/Dec": "Rektaszension/Dekklination",
```

Comment:
sicher ohne Doppel-K

Reply:
Du hast natürlich recht. Danke für den Hinweis.
Ich habe es in Ra / Dek umbenannt.